README.md

```diff
@@ -47,8 +49,6 @@ Seed the database
 php artisan db:seed
 ```
 
-Set db credentials in `.env` and run the project.
-
 For Login
```

Comment:
Add the login details before the development heading i.e after the project serve and its url.